Layout scale bar review (#5450)
qgis/QGIS-Documentation: docs/user_manual/print_composer/composer_items/composer_scale_bar.rst

     segments, with horizontal lines through alternating segments
   * **Numeric**, where the scale ratio is printed (e.g., ``1:50000``).
#. Set properties as appropriate

Units
-----

The :guilabel:`Units` group of the scale bar :guilabel:`Item Properties` panel
provides the functionalities to set the units of display and some text formatting
(see figure_layout_scalebar_units_):

.. _figure_layout_scalebar_units:

.. figure:: img/scalebar_units.png
   :align: center

   Scale Bar Units group

* Select the units you want to use with :guilabel:`Scalebar units`. There are
  many possible choices: **Map Units** (the default one), **Meters**, **Feet**,
  **Miles** or **Nautical Miles**... and some derivatives. Units conversion
  is handled automatically.
* The :guilabel:`Label unit multiplier` specifies how many scale bar units per
  labeled unit. Eg, if your scale bar units are set to "meters", a multiplier of
  1000 will result in the scale bar labels in "kilometers".
* The :guilabel:`Label for units` field defines the text used to describe the
  units of the scale bar, eg ``m`` or ``km``. This should be matched to reflect
  the multiplier above.

> * Press :guilabel:`Customize`next to :guilabel:`Number format` to have control

Author: ```suggestion
* Press :guilabel:`Customize` next to :guilabel:`Number format` to have control
```

  over all the formatting properties for the numbers in the scale bar, including
  thousand separators, decimal places, scientific notation, etc.
  (see :ref:`number_formatting` for more details).
  Very useful in the case of making maps for audiences outside of the current

>   QGIS locale, or when you'd just prefer to vary the style from the locale

Reviewer: ```suggestion
  QGIS locale, or when you would like to vary the style from the locale
```

  defaults (e.g. adding thousands separators when the locale default is to hide

>   them)

Reviewer: ```suggestion
  them).
```

Segments
--------

The :guilabel:`Segments` group of the scale bar :guilabel:`Item Properties` panel
provides the functionalities to configure the number and size of segments and
subdivisions (see figure_layout_scalebar_segments_):

.. _figure_layout_scalebar_segments:

.. figure:: img/scalebar_segments.png
   :align: center

   Scale Bar Segments group

> * You can define how many :guilabel:`Segments` will be drawn on the left and on

Reviewer: ```suggestion
* You can define how many :guilabel:`Segments` that will be drawn on the left and on
```

Author: not convinced, here.

Reviewer: Including the "that" makes the sentence easier to digest.

  drawing using :ref:`fill symbols properties <vector_fill_symbols>`
  (color, opacity, patterns, effects...) --- for *Single Box*, *Double Box*
  and *Hollow* styles
* :guilabel:`Line style` of the scale bar drawing using :ref:`line symbols
  properties <vector_line_symbols>` (color, stroke, join, cap style, patterns,
  effects...) --- for all but *Numeric* style
* :guilabel:`Division style` and :guilabel:`Subdivision style` respectively
  for division and subdivision segments in *Line Ticks Up*, *Line Ticks Middle*
  and *Line Ticks Down* scale bar styles using :ref:`line symbols properties
  <vector_line_symbols>` (color, stroke, join, cap style, patterns, effects...)
* :guilabel:`Alignment` puts text on the left, center or right side of the
  frame (only for *Numeric* scale bar style)
* :guilabel:`Font` to set the :ref:`properties <text_format>`
  (size, font, color, letter spacing, shadow, background...) of the scale bar
  label.

Since most of the display properties of the scale bar rely on symbols whose
properties can be data-defined, it's possible to render data-defined scale bars.

**Example**: The following code applied to the bold property of the scale labels

> will display texts in bold when they are a multiple of 500:

Reviewer: ```suggestion
will display numbers in bold when they are a multiple of 500:
```

Author: humm.. I used "text" because I noticed that the unit was also in bold but testing more, I realize it uses the last number property. A feature request probably to configure units font separately.
